src/Queue.js

```diff
@@ -20,33 +20,35 @@ class Queue {
     // Also sets the input process's parent queue to this queue
     // Return the newly added process
     enqueue(process) {
-        
+        this.processes.push(process);
+        process.setParentQueue(this);
+        return process;
```

Comment:
This works 100% fine but you can do this in 2 lines instead of 3.
`process.setParentQueue(this);`
`return this.process.push(process);`
Either works fine just bringing up an alternate solution for this section